apps/web/components/eventtype/AIEventController.tsx

```diff
@@ -157,68 +206,89 @@ const AISettings = ({ eventType }: { eventType: EventTypeSetup }) => {
           <Label>{t("your_phone_number")}</Label>
           <Controller
             name="aiPhoneCallConfig.yourPhoneNumber"
-            render={({ field: { onChange, value } }) => {
+            render={({ field: { onChange, value, name }, fieldState: { error } }) => {
               return (
-                <PhoneInput
-                  required
-                  placeholder={t("your_phone_number")}
-                  id="aiPhoneCallConfig.yourPhoneNumber"
-                  name="aiPhoneCallConfig.yourPhoneNumber"
-                  value={value}
-                  onChange={(val) => {
-                    onChange(val);
-                  }}
-                />
+                <div>
+                  <PhoneInput
+                    required
+                    placeholder={t("your_phone_number")}
+                    id="aiPhoneCallConfig.yourPhoneNumber"
+                    name="aiPhoneCallConfig.yourPhoneNumber"
+                    value={value}
+                    onChange={(val) => {
+                      onChange(val);
+                    }}
+                  />
+                  {error?.message && <ErrorMessage message={error.message} fieldName={name} />}
+                </div>
               );
             }}
           />
 
+          <>
+            <RadioGroup.Root
+              defaultValue={eventType?.aiPhoneCallConfig?.templateType ?? "CUSTOM_TEMPLATE"}
+              onValueChange={(val) => {
+                formMethods.setValue("aiPhoneCallConfig.templateType", val, { shouldDirty: true });
+              }}>
+              <div className="flex gap-2">
+                <div className="flex flex-1 rounded-lg border p-4">
+                  <RadioGroup.Item
+                    id="CHECK_IN_APPOINTMENT"
+                    key="CHECK_IN_APPOINTMENT"
+                    value="CHECK_IN_APPOINTMENT"
+                    className="min-w-4 bg-default border-default flex h-4 w-4 cursor-pointer items-center rounded-full border focus:border-2 focus:outline-none ltr:mr-2 rtl:ml-2 ">
+                    <RadioGroup.Indicator className="after:bg-inverted relative flex h-4 w-4 items-center justify-center after:block after:h-2 after:w-2 after:rounded-full" />
+                  </RadioGroup.Item>
+                  <label className="h-full" htmlFor="CHECK_IN_APPOINTMENT">
+                    <h2 className="font-semibold leading-tight">{t("check_in_assistant")}</h2>
+                    <p className="text-subtle mt-2">{t("check_in_assistant_description")}</p>
```

Comment:
font size looks too big, I feel like `text-sm for` both heading and description would be better

Reply:
Okay. I'll raise a new PR for this